event/register_oracle_event.go

```diff
@@ -51,9 +59,93 @@ func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
 
 	err = sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, nodePubKeyHash[:], *e.reactor.EnclaveInfo())
 	if err != nil {
+		msgVoteOracleRegistrationNo, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, types.VOTE_OPTION_NO, e.reactor.OraclePrivKey())
+		if err != nil {
+			return err
+		}
+
+		txBytes, err := generateTxBytes(msgVoteOracleRegistrationNo, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), e.reactor.TxBuilder())
+		if err != nil {
+			return err
+		}
+
+		err = broadCastTx(e.reactor.GRPCClient(), txBytes)
+		if err != nil {
+			return err
+		}
 		return err
 	}
 
-	// TODO: Executing Vote Txs
+	msgVoteOracleRegistrationYes, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, types.VOTE_OPTION_YES, e.reactor.OraclePrivKey())
+	if err != nil {
+		 return err
+	}
+
+	txBytes, err := generateTxBytes(msgVoteOracleRegistrationYes, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), e.reactor.TxBuilder())
+	if err != nil {
+		return err
+	}
+
+	err = broadCastTx(e.reactor.GRPCClient(), txBytes)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// makeOracleRegistrationVote makes a vote for oracle registration with VOTE_OPTION
+func makeOracleRegistrationVote(uniqueID, voterAddr, votingTargetAddr string, voteOption types.VoteOption, oraclePrivKey []byte) (*types.MsgVoteOracleRegistration, error) {
+	registrationVote := &types.OracleRegistrationVote{
+		UniqueId:               uniqueID,
+		VoterAddress:           voterAddr,
+		VotingTargetAddress:    votingTargetAddr,
+		VoteOption:             voteOption,
+		EncryptedOraclePrivKey: oraclePrivKey,
+	}
+
+	key := secp256k1.PrivKey{
+		Key: oraclePrivKey,
+	}
+
+	bytes, err := registrationVote.Marshal()
+	if err != nil {
+		return nil, err
+	}
+
+	sign, err := key.Sign(bytes)
+	if err != nil {
+		return nil, err
+	}
+
+	msgVoteOracleRegistrationNo := &types.MsgVoteOracleRegistration{
+		OracleRegistrationVote: registrationVote,
+		Signature:              sign,
+	}
+
+	return msgVoteOracleRegistrationNo, nil
+}
+
+// generateTxBytes
+func generateTxBytes(msgVoteOracleRegistration *types.MsgVoteOracleRegistration, privKey cryptotypes.PrivKey, conf *config.Config, txBuilder *panacea.TxBuilder) ([]byte, error) {
+	defaultFeeAmount, _ := sdk.ParseCoinsNormalized(conf.Panacea.DefaultFeeAmount)
+	txBytes, err := txBuilder.GenerateSignedTxBytes(privKey, conf.Panacea.DefaultGasLimit, defaultFeeAmount, msgVoteOracleRegistration)
+	if err != nil {
+		return nil, err
+	}
+
+	return txBytes, nil
+}
+
+// broadCastTx
+func broadCastTx(grpcClient *panacea.GrpcClient, txBytes []byte) error {
+	resp, err := grpcClient.BroadcastTx(txBytes)
+	if err != nil {
+		return err
+	}
+
+	if resp.TxResponse.Code != 0 {
+		return fmt.Errorf("register oracle vote failed: %v", resp.TxResponse.RawLog)
+	}
```

Comment:
Like `register-oracle`, let's print the info log about transaction.
```suggestion
        }
        log.Infof("MsgVoteOracleRegistration transaction succeed. height(%v), hash(%s)", resp.TxResponse.Height, resp.TxResponse.TxHash)
```